tests/monitoring/request_collector/test_server.py

```diff
@@ -93,3 +95,11 @@ def stored_mr_after_proccessing(amount, nonce):
 
     # update without higher nonce must succeed
     assert stored_mr_after_proccessing(amount=2, nonce=2).balance_hash != initial_hash
+
+
+def test_request_collector_doesnt_crash_with_invalid_messages(request_collector):
+    # We want to test that the request collector does not crash,
+    # in case an assertion on the MonitorRequest fails
+    # In theory, the collector crashed before, when on_monitor_request raised an AssertionError
```

Comment:
>  In theory, the collector crashed before, when on_monitor_request raised an AssertionError
	
Before what? The comment will stay in the code, so if you're referring to this PR, this won't be understood after this PR is merged.

Reply:
ah that is a valid point hehe changed it